internal/resolver/resolver.go

```diff
@@ -57,7 +57,11 @@ func (r *Resolver) Resolve(reqs []*chart.Dependency, repoNames map[string]string
 	locked := make([]*chart.Dependency, len(reqs))
 	missing := []string{}
 	for i, d := range reqs {
-		constraint, err := semver.NewConstraint(d.Version)
+		v := d.Version
+		if len(strings.TrimSpace(v)) == 0 {
```

Comment:
`if len(strings.TrimSpace(v)) == 0 && strings.HasPrefix(d.Repository, "file://") {`

Reply:
Is just `file://` optional?

Reply:
That is my understanding. I can't confirm. https://helm.sh/docs/helm/helm_dependency/